addons/website/static/src/systray_items/translate_website.xml

```diff
@@ -2,10 +2,12 @@
 <templates xml:space="preserve">
 
     <t t-name="website.TranslateWebsiteSystray" owl="1">
-        <div class="o_translate_website_container d-none d-md-block">
-            <a href="#" accesskey="t" t-on-click="startTranslate" class="btn btn-primary d-flex align-items-center h-100 rounded-0 border-0 fw-normal">
-                <span t-if="websiteContext.translation" class="fa fa-refresh fa-spin me-2"/>
-                Translate
+        <div class="o_menu_systray_item o_translate_website_container d-none d-md-block">
+            <a href="#" accesskey="t" t-on-click="startTranslate" class="btn btn-primary d-flex align-items-center h-100 px-0 rounded-0 border-0 fw-normal">
+                <span class="o_nav_entry">
```

Comment:
So the new commit is adding 2 `<span/>`, changing the DOM tree.
It's not possible in stable, clients are already starting in Odoo 16.4 atm, so 16.3 is really not possible IMO.
Unless @qsm-odoo has another opinion?

Reply:
@rdeodoo it's in static xml, so it should be ok to add a new node :wink: